plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginCommandHandler.java

```diff
@@ -19,40 +15,33 @@
 
 public class GoogleLoginCommandHandler extends AbstractHandler implements IElementUpdater {
 
+  private IGoogleLoginService getGoogleLoginService() {
```

Comment:
Ditto about avoiding _PlatformUI.getWorkbench()_ made in [_StandardDeployCommandHandler_](https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/486/files#r73461563)